src/utils/osUtils.ts

```diff
@@ -88,3 +89,15 @@ export async function getDockerOSType(context: IActionContext): Promise<DockerOS
         return info.OSType;
     }
 }
+
+export async function getContainerOSType(containerId: string, context: IActionContext): Promise<ContainerOSType> {
```

Comment:
@ravipal I just realized; I think `getDockerOSType` is sufficient for this. The Docker server OS must match the container OS.

Reply:
Now using DockerOS instead of container OS and it will break only LOCW.